images/openshift-enterprise-haproxy-router-base.yml

```diff
@@ -0,0 +1,23 @@
+mode: wip
+base_only: true
+content:
+  source:
+    dockerfile: Dockerfile.rhel
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+        fallback: master
+      url: git@github.com:openshift/router.git
+    path: images/router/base
```

Comment:
There's nothing under this path except the Dockerfile. I don't think you want this as build context.